code_schemes/s01e13.json

```
@@ -201,6 +201,70 @@
         "StringValue": "rumour_stigma_misinfo_misinfo_on_status",
         "VisibleInCoda": true
       },
+      {
+        "CodeID": "code-2c40cdf0",
+        "CodeType": "Normal",
+        "DisplayText": "lack of healthcare",
+        "NumericValue": 22,
+        "StringValue": "lack_of_healthcare",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-e9a6db7e",
+        "CodeType": "Normal",
+        "DisplayText": "lack of support",
+        "NumericValue": 23,
+        "StringValue": "lack_of_support",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-a7c8e0ce",
+        "CodeType": "Normal",
+        "DisplayText": "discrimination",
+        "NumericValue": 24,
+        "StringValue": "discrimination",
+        "VisibleInCoda": true
+      },
+      {
+        "CodeID": "code-95a534e6",
+        "CodeType": "Normal",
+        "DisplayText": "lack of access of basic needs",
```

```suggestion
        "DisplayText": "lack of access to basic needs",
```
I think this reads slightly better, but don't mind leaving it if RDA prefers

---

Thanks.